src/mon/MgrMonitor.cc

```diff
@@ -346,6 +337,12 @@ void MgrMonitor::tick()
     return;
 
   const auto now = ceph::coarse_mono_clock::now();
+  if (is_active() &&
+      chrono::duration_cast<chrono::seconds>(now - last_sent_digest).count() >
+      g_conf->mon_mgr_digest_period) {
```

Comment:
Hmm, this is going to give pretty irregular messages if mon_mgr_digest_period is not a multiple of the tick period.